app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,29 @@
+class CommentsController < ApplicationController
+
+  def show
+    @comment = Comment.find(params[:id])
+  end
+
+  def new
+    @comment = Comment.new
+  end
+
+  def create
+    @comment = Comment.new(comment_params)
+    @comment.job_id = params[:job_id]
+    @job = @comment.job
+    @company = @job.company
+    if @comment.save
+      flash[:success] = "Your created a new comment!"
+      redirect_to company_job_path(@company, @job)
+    else
+    render :new
```

Comment:
if a user enters invalid data, we want to show them a nice error message - don't forget to handle your errors!